.travis.yml

```diff
@@ -3,9 +3,8 @@ python:
   - "2.7"
   - "3.6"
 install:
-    - pip install -r requirements.txt
-    - pip install -r requirements-test.txt
-    - pip install paver
+    - pip install pipenv
+    - pipenv install --dev
 script: paver test
```

Comment:
I think this should be ran via `pipenv run paver test` and paver should become a dev dependency.

Reply:
I think we can remove paver: https://github.com/pypa/pipenv/wiki#custom-run-commands cc @Almad